packages/react/src/internal/hooks/useVideoData/useVideoData.js

```diff
@@ -16,8 +16,13 @@ import { VideoPlayerAPI } from '@carbon/ibmdotcom-services';
  * @returns {*} JSX Object
  */
 function useVideoData(type, videoId) {
-  const [videoTitle, setVideoTitle] = useState([{ title: '', key: 0 }]);
+  const [videoTitle, setVideoTitle] = useState([
+    { title: '', duration: '', key: 0 },
+  ]);
 
+  useEffect(() => {
+    getVideoData();
+  }, [getVideoData, type]);
```

Comment:
Hey @asudoh I noticed the order of the `useEffect` placement was causing issues whenever I changed the CTA type to `video`. There would be an infinite re-rendering of the CTA component. I saw that you made the change to move the `useEffect` to fix transpilation. I wonder if there is another way to get around the issue?

Reply:
@annawen1 My apologies for the breakage. Yes, ensuring `getVideoData` variable declared at the top fixes the problem and thank you for doing this!